### PR TITLE
Update 2018-fortaleza.yml

### PR DESCRIPTION
**Pull request by gb8may**
Add new member on the team: Jairo Junior
Update informations about members: Profile picture, Bio, Role..

*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2018] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: data/events/2018-fortaleza.yml [hunk 1]
```diff
@@ -62,19 +62,42 @@ nav_elements: # List of pages you want to show up in the navigation of your page
...
 team_members: # Name is the only required field for team members.
   - name: "Mayara Gouveia"
-    twitter: "devopsdays"
+    role: "Jr DevOps Engineer"
+    bio: "Sysadmin | DevSecOps | Information Security Specialist. Data Cabling System Furukawa Certificate, Fluke Networks, CompTIA Network +, Ethical Hacking Foundation and Fortinet Network Security Essentials 1. Network Forensics Expert. Guest Speaker in spare times. Mac OSX user. Experience with DevSecOps; AWS Cloud; GNU / Linux; Network routing and switching; Structured Cabling; Virtualization; IT services; WSO2."
     linkedin: "https://www.linkedin.com/in/gb8may/"
     website: "https://mayara.gouveia.info"
-    github: "devopsdays"
+    github: "https://github.com/gb8may"
+    image: "mayara-gouveia.jpg"
```
Comment (gomex) [inline data/events/2018-fortaleza.yml]: Essas imagens precisam estar na pasta de imagens do seu evento.

Veja esses exemplos:

Olha o yaml do evento: https://github.com/devopsdays/devopsdays-web/blob/master/data/events/2018-beijing.yml

Olha a pasta de imagens: https://github.com/devopsdays/devopsdays-web/tree/master/static/events/2018-beijing/organizers

File: data/events/2018-fortaleza.yml [hunk 2]
```diff
@@ -62,19 +62,42 @@ nav_elements: # List of pages you want to show up in the navigation of your page
...
   - name: "Mauricio Harley"
+    role: "Senior Network/Security Engineer"
+    bio:
     linkedin: "https://www.linkedin.com/in/mauricioharley"
     website: "https://itharley.com"
+    twitter:
+    image: "mauricio-harley.jpg"
```
Comment (gomex) [inline data/events/2018-fortaleza.yml]: Mesma coisa aqui.

File: data/events/2018-fortaleza.yml [hunk 3]
```diff
@@ -62,19 +62,42 @@ nav_elements: # List of pages you want to show up in the navigation of your page
...
   - name: "Ana Carla Cavalcante"
+    role:
+    bio:
     linkedin: "https://www.linkedin.com/in/ana-carla-cavalcante-62a40725/"
+    image: "ana-carla.jpg"
```
Comment (gomex) [inline data/events/2018-fortaleza.yml]: E aqui...

File: data/events/2018-fortaleza.yml [hunk 4]
```diff
@@ -62,19 +62,42 @@ nav_elements: # List of pages you want to show up in the navigation of your page
...
   - name: "Jorge Santos"
+    role:
+    bio:
     linkedin: "https://www.linkedin.com/in/jorge-luiz-ba946329/"
+    image: "jorge-santos.jpg"
```
Comment (gomex) [inline data/events/2018-fortaleza.yml]: Essas imagens precisam estar na pasta de imagens do seu evento.

Veja esses exemplos:

Olha o yaml do evento: https://github.com/devopsdays/devopsdays-web/blob/master/data/events/2018-beijing.yml

Olha a pasta de imagens: https://github.com/devopsdays/devopsdays-web/tree/master/static/events/2018-beijing/organizers

File: data/events/2018-fortaleza.yml [hunk 5]
```diff
@@ -6,7 +6,7 @@ description: "Devopsdays is coming to Fortaleza!" # Edit this to suit your prefe
 ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
 
 # All dates are in unquoted 2018-MM-DD, like this:   variable: 2016-01-05
-startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+startdate: 2018-10-20 # The start date of your event. Leave blank if you don't have a venue reserved yet.
 enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (gomex) [inline data/events/2018-fortaleza.yml]: Se será um dia só, coloque a mesma data no enddate.

File: data/events/2018-fortaleza.yml (hunk 4 above)
Comment (gb8may) [reply data/events/2018-fortaleza.yml]: Eu não consegui criar o diretorio organizers por aqui. Vou tentar novamente.

**Pull request closed without merging**